doc/reference/configuration/configuration_reference.rst

```diff
@@ -2412,11 +2458,13 @@ The ``process`` defines configuration parameters of the Tarantool process in the
     | Default: false
     | Environment variable: TT_PROCESS_BACKGROUND
 
-
 .. _configuration_reference_process_coredump:
 
 .. confval:: process.coredump
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Whether to create coredump files.
```

Comment:
Just tested it. It actually works on my system, wow!

At least, it works if I just run tarantool without daemonization and without changing an effective user.

Usually an administrator needs to call `ulimit -c unlimited` before running tarantool process to get coredumps (or set corresponding options in systemd's unit file). If `process.coredump` is enabled, tarantool set the corresponding resource limit by itself and the admin don't need to call `ulimit -c unlimited`.

As I see from the [code](https://github.com/tarantool/tarantool/blob/01915e6d34bbe6c6bdd5042f8a3d1f4333c08975/src/main.cc#L381-L396), the option also sets the `dumpable` attribute, which is enabled by default, but may be dropped in some circumstances (according to [man 2 prctl](https://man7.org/linux/man-pages/man2/prctl.2.html), see `PR_SET_DUMPABLE`).

It is interesting that it is [marked](https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-basic-coredump-deprecated) as deprecated in the box.cfg reference. I don't know why. We found it useful and included into the new declarative configuration.

You can include some of these details into the option description. I think it may be useful.

Reply:
> It is interesting that it is [marked](https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-basic-coredump-deprecated) as deprecated in the box.cfg reference. I don't know why. We found it useful and included into the new declarative configuration.

I guess that we should undeprecate it in the box.cfg reference for consistency with the declarative configuration reference.

If we meet the problem that was the reason for the deprecation again, then we return here back. Now it seems working and useful.